### PR TITLE
Smartcard updates

### PR DESCRIPTION
**Pull request by akallabeth**
* Implements missing functions for newer smartcard channel behaviour
* Adds command line option to set client build number

### CODE REVIEW

**Comment by akallabeth** [comment]
@ondrejholy So, I´ve updated the smartcard channel to the current spec.
Did not have time to test extensively, but now it connects also with newer `client build number`

File: channels/smartcard/client/smartcard_main.c [hunk 1]
```diff
@@ -428,7 +428,7 @@ static UINT smartcard_process_irp(SMARTCARD_DEVICE* smartcard, IRP* irp)
 			case SCARD_IOCTL_ISVALIDCONTEXT:
 			case SCARD_IOCTL_CANCEL:
 			case SCARD_IOCTL_ACCESSSTARTEDEVENT:
-			case SCARD_IOCTL_RELEASESTARTEDEVENT:
+			case SCARD_IOCTL_RELEASETARTEDEVENT:
 				asyncIrp = FALSE;
```
Comment (hardening) [inline channels/smartcard/client/smartcard_main.c]: Are you sure about this renaming ?

Comment (akallabeth) [reply channels/smartcard/client/smartcard_main.c]: this way it conforms to `[MS-RDPESC]` naming.

**Comment by akallabeth** [comment]
@freerdp-bot test

**Pull request merged**